site/src/components/PageHeader.tsx

```diff
@@ -64,7 +65,15 @@ const PageHeader: React.SFC<PageHeaderProps> = ({
             </div>
           )}
           <div className={styles.mainSection}>
-            <h1 className={styles.headingText}>{headingText}</h1>
+            <div className={styles.headingTextContainer}>
+              <Heading
+                variant={headingOnly ? "display-0" : "heading-1"}
+                tag="h1"
+                classNameAndIHaveSpokenToDST={styles.headingTextOverrides}
```

Comment:
This is because the color needed to be overridden to white. We are looking into extending the API during cooldown.